assets/style.css

```diff
@@ -0,0 +1,603 @@
+  z-index: 10;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.nav-bar {
+  height: 100%;
+  align-items: center;
+}
+
+ul {
+  list-style: none;
+}
+
+.hamburger-search {
+  flex: 1;
+  margin-left: 5px;
+  height: 100%;
+}
+
+.hamburger-search li .fa-bars {
+  padding-right: 30px;
+  margin-left: -25px;
+}
+
+.nav-logo {
+  flex: 1;
+  height: 100%;
+}
+
+.heading-img img {
+  width: 100%;
+}
+
+.space-astronomy img,
+.editor-pick img {
+  width: 290px;
+  height: 200px;
+}
+
+.telescope-img img {
+  display: block;
+  width: 100%;
+}
+
+.footer img {
+  display: block;
+  width: 200px;
+  height: 60px;
+  margin-top: -10px;
+  margin-bottom: -5px;
+  padding: 0;
+}
+
+.subscribe-img img {
+  width: 20px;
+  height: 20px;
+}
+
+.nav-logo a img {
+  display: block;
+  height: 50px;
+  margin-top: -5px;
+}
+
+.first-row ul li figure img,
+.second-row ul li figure img {
+  height: 130px;
+  width: 215px;
+}
+
+.nav-logo a img:hover {
+  cursor: pointer;
+}
+
+.subscribe-login {
+  margin-right: 20px;
+  height: 100%;
+}
+
+.login {
+  float: right;
+}
+
+.btn {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 11px;
+  background-color: #567b95;
+  color: #fff;
+  padding: 11px 12px 8px;
+  line-height: 11px;
+  border-radius: 3px;
+  border: 1px solid rgba(50, 104, 145, 0);
+}
+
+.subscribe {
+  margin-right: 7px;
+}
+
+/* Main */
+main {
+  margin-top: 40px;
+}
+
+.heading-img {
+  width: 85%;
+  margin: 0 auto;
+}
+
+.heading-img h1 {
+  margin: auto;
+  font-size: 40px;
+  width: 600px;
+  font-style: italic;
+  padding: 4rem 0 1rem 0;
+  font-weight: 900;
+}
+
+.heading-img figcaption {
+  font-size: 15px;
+  color: rgb(44, 43, 43);
+  line-height: 20px;
+  width: 75%;
+}
+
+.telescope-img span {
+  font-size: 15px;
+}
+
+.subscribe-img span {
+  font-weight: 900;
+}
+
+.heading-img figcaption span {
+  font-size: 13px;
+  line-height: 18px;
+}
+
+.width-class {
+  width: 40%;
+  margin: 0 auto;
+}
+
+.content {
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 20px;
+  line-height: 30px;
+  color: #333;
+  font-weight: 400;
+}
+
+.main-content .author-name {
+  margin-top: 50px;
+  margin-bottom: 5px;
+}
+
+.extra-info a {
+  text-decoration: underline;
+}
+
+.space-astronomy a {
+  grid-column: 1/-1;
+  font-size: 16px;
+  line-height: 22px;
+  color: #000;
+  font-weight: 700;
+}
+
+.most-popular a {
+  font-size: 19px;
+  line-height: 23px;
+  font-weight: 500;
+  color: #000;
+  display: inline-block;
+  padding: 10px 0;
+  margin: 10px 0;
+  border-top: 1px solid rgba(211, 204, 204, 0.822);
+  text-decoration: none;
+}
+
+.footer-bottom a {
+  display: inline-block;
+  text-decoration: none;
+  padding: 0 5px;
+  font-size: 11px;
+  color: #666;
+  font-weight: 400;
+}
+
+.footer-heading a {
+  font-size: 13px;
+  text-decoration: none;
+  color: #567b95;
+}
+
+.editor-pick-aside a {
+  color: #333;
+  text-decoration: none;
+  font-size: 14px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-z{
  color: #333;
}
```
**Html file**

```
<div class="color-z">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_